core/modules/catalog/store/product/actions.ts

```diff
@@ -684,7 +685,7 @@ const actions: ActionTree<ProductState, RootState> = {
     if (product && product.category_ids) {
       let currentCategory = rootGetters['category-next/getCurrentCategory'] // use current category, if set
       if (!currentCategory || !currentCategory.id || !product.category_ids.includes(currentCategory.id.toString())) {
-        const categoryFilters = Object.assign({ 'id': product.category_ids }, config.entities.category.breadcrumbFilterFields)
+        const categoryFilters = Object.assign({ 'id': [...product.category_ids] }, cloneDeep(config.entities.category.breadcrumbFilterFields))
```

Comment:
```suggestion
        const categoryFilters = cloneDeep(Object.assign({ 'id': product.category_ids },config.entities.category.breadcrumbFilterFields))
```

Reply:
I'm not sure about this suggestion. If we know that product.category_ids is a 1-dimension array, then it's sufficient to use destructuring. This way we're cloning a smaller object.